snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-1

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Here's the diff between the 2 versions, I hope this diff will help us spot any errors easier (but I hope there would be none 😛 ).

```diff
diff --git a/snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-0 b/snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-1
index 9a70275191..1b1fc0455d 100644
--- a/snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-0
+++ b/snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-1
@@ -5,7 +5,7 @@
     "vendor": "com.metabase",
     "name": "account",
     "format": "jsonschema",
-    "version": "1-0-0"
+    "version": "1-0-1"
   },
   "type": "object",
   "properties": {
@@ -14,7 +14,28 @@
       "type": "string",
       "enum": [
         "new_user_created",
-        "new_instance_created"
+        "new_instance_created",
+        "db_connection_banner_click"
+      ],
+      "maxLength": 1024
+    },
+    "version": {
+      "description": "String describing the version of database connection banner we're on",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 1024
+    },
+    "link": {
+      "description": "String identifying the link where the user clicked",
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "nav",
+        "help"
       ],
       "maxLength": 1024
     }
```